pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.kohsuke</groupId>
     <artifactId>pom</artifactId>
-    <version>14</version>
+    <version>19</version>
```

Comment:
Ideally we should start migrating it to https://github.com/jenkinsci/pom.
IIUC there is no other things excepting Plugin site release and code signing there. WDYT @jglick ?

Reply:
Not sure. Technically this is a non-Jenkins component.

Reply:
It's something we could also fix. Stapler is a critical part of Jenkins && it's not extremely  popular elsewhere, so moving it && integrating common Jenkins QA/review flow would not hurt. It's something we were discussing multiple times, but maybe it belongs to the ML now.

Reply:
Basically that would mean not releasing it to Central any more. Fine with me, but needs some discussion.